AutomatedNotebooks-Manager.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<p>Just wondering about if something fails in the notebook you'll leave a dangling resource (although I guess it will get cleaned up when the process exits)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>